### ISSUE TITLE
Authors should provide their ORCID iD

### ISSUE THREAD

**Issue by pigsonthewing** [alphagov/tech-and-data-standards#34]
Any named individual who writes a government document, standard or code should provide an ORCID iD, which should be published alongside their byline, and in associated metadata.

ORCID iDs are unique identifiers. The use of an ORCID disambiguates two authors with the same (or similar) names; and identifies the work of one person under a variety of names (for example because of differing use of initials, misspellings, name changes, or differing transliterations).

Individuals register and own their ORCID record; it goes with them when they change jobs, or write for other publishers. The record can include details of education, employment, funding and works authored, each of which can be made public or kept private.

Publishers, employers, funders and other bodies can incorporate ORCID into their back-end systems.  APIs are available publicly and to paid members of ORCID or a local consortium. Some mandate that authors or people receiving funding must provide an ORCID iD. Others make it optional.

ORCID is a non-profit organisation, and public ORCID data is available under an open licence.

As an organisation, the government should encourage, and where appropriate mandate, the use of ORCID iDs, and should include parameters for them in relevant forms, and databases.

For more info, see http://orcid.org/ or https://en.wikipedia.org/wiki/ORCID

[My ORCID is in my Github profile.]

**Comment by edent** [comment]
A really interesting idea. Thanks Andy. Two main questions.

1. Is ORCID tied to a person or a position?
   * If a statement is issued by the Minister of Magic  - is it written by Cornelius Fudge (ORCID 123456) or by the **Office of the Minister**?
   * A document is written by *multiple* people - all the Aurors in the Ministry - but *published* by the Chief Auror.  Is it done in the name of the position, of Mad Eye Moody (ORCID 56789), or all the authors?
   * We will need to understand if individuals *want* to be associated with the documents they publish officially.
2. What problem does this solve?
   * What are the external needs that this satisfies? Are government documents regularly cited in the same way as academic documents?
   * Do people (internal or external) regularly need to contact the *specific author* of a form? Or should they be contacting the department which issued it?
   * How are documents signed? What is to stop me putting Albus Dumbledore's ORCID ID on a document written by Severus Snape?

If you - or anyone else - has any insight, we'd be very interested in reading it.

**Comment by gbilder** [comment]
1) ORCID focuses on individual contributors. Not corporate authorship. Specifically on `1.3` (do individuals _want_ to be associated with the documents they publish officially)-  there are two things to note. First is that it isn't all-or-nothing- ORCID can support pseudo-anonymity. The debates about the role real names and openness are fraught. I don't think ORCID is trying to legislate in favour of real names. Rather, their focus is on helping to reduce ambiguity when understanding the provenance of content is important. It can be useful to know that the *same* anonymous person was the author of several reports. Second, the decision to either credit individual authors or use corporate authorship is usually down to the culture and traditions of the discipline/industry. 

2.  2.1 Yes- government documents are regularly cited. This is certainly true in the academic literature. But is seems reasonably likely that they are cited by other government documents, NGOs/IGOs, Standards bodies, ThinkTanks, the media, etc.
2.2 I'm not sure how much of the issue is about contacting authors. The important thing here seems to be understanding provenance. 
2.3 What's to stop Severus from doing that now? ORCID is trying to address the name ambiguity problem, not the honesty or fraud problems. At least you'd be sure which specific Albus Dumbledore Severus was trying to to impersonate. 

Final added benefit is that He-who-can-not-be-named can at least get an ORCID.

-G
http://orcid.org/0000-0003-1315-5960

**Comment by edent** [comment]
Thanks for the info @gbilder 

I think the issue here is that many Government documents are written by the *office* rather than an individual.

I also find the lack of authentication troubling.  Suppose I publish a paper about astrology with Albert Einstein's ORCID iD, how can a user tell if it genuine or not?

@pigsonthewing are there some *specific* documents which you think would benefit from having ORCID metadata?  I think it would be helpful to have some examples of where this could be useful.

Thanks all.

https://orcid.org/0000-0002-9265-9069

**Comment by pigsonthewing** [comment]
While it is true that "many" Government documents are written by the office rather than an individual, not all are, and I specifically address the latter case.

While Albert Einstein can't have an ORCID iD (they have to be registered by a living person), your hypothetical example would be confirmed by him including the paper on his ORCID profile. What's to stop you using his _name_ on the paper? As Geoffrey says, "ORCID is trying to address the name ambiguity problem, not the honesty or fraud problems."

Examples of publications with named individuals as authors, where ORCID iDs would be beneficial,  include:

https://www.gov.uk/government/uploads/system/uploads/attachment_data/file/584665/summary-local-authority-insight-research-wave-32.pdf 

which has _three_ authors; and GDS blog posts.

**Comment by edent** [comment]
Interesting.  So I think we can restrict this just to publications where named individuals (rather than positions) *want* to promote their ORCID iD.

Are their any standards for *how* the metadata should be included in a document?  I took a look through the ORCID site, but couldn't see anything obvious.

For HTML, based on the [WordPress plugin](https://github.com/ServerDotBiz/orcid/), it seems enough to link to ORCID using the `rel="author"` attribute.  Is that documented anywhere?

For PDFs - I can't find anything about author metadata.  Is there a definitive answer to this?

**Comment by PeterParslow** [comment]
Interestingly, this came to my attention as a new "Open Standards Challenge", as per those handled at https://standards.data.gov.uk/, and I was going to ask "what's the problem?" i.e. "problems with government services might be solved by open standards".

Re-phrasing it that way, instead of starting with the answer, may mean asking "How can I uniquely identify the author of a government document?" That would then divide into organisations and people, and (given that it's a government document), the organisations part would most likely be answered by reference to a register (if only there was an easy way to find the 'register of registers' - but that's a separate problem, already raised with GDS).

And it may be that ORC iD is the correct answer for the 'people' part - the open process would give others the opportunity to mention standards they are aware of.

**Comment by Lawrence-G** [comment]
It’s OK to suggest a candidate standard as a solution to a problem at the outset of a challenge. The Board agreed last year that
“where there are already well-proven use cases. The fast track would allow mature suggestions to start at the 'proposal' stage with a standard ready for assessment. For challenges with an already identified standard the ‘Suggestion’ and ‘Response’ phases would be combined." see the [OCDS](https://standards.data.gov.uk/challenge/open-contracting-data-standard) challenge from last year.
The case for the standard would still require a user need, as described in Peter's comment

From an open standards point of view - this is a number, issued by an organisation. We have adopted standards before where the output in the form of a code is what is used rather than the specification (e.g. country codes). I feel however ORCID would be difficult to access as a standard
in our process using the [core questions](https://standards.data.gov.uk/core-assessment-questions)  as many Qs would have to be deemed not applicable  

Orchid is not, I think, a standard organisation that would be recognised in our assessment

I’m not critical of ORCID or the suggestion that an ID code for publications is of use. I’m not sure that it fits our remit in open standards. This is the place to discuss these issues though.
Any thoughts on that?

**Comment by edent** [comment]
Thank you all for your contributions to this discussion.

I've spoken to our web publishing team. Currently, authors are able to link their posts to any personal ID source they wish - LinkedIn, Twitter, and their own blog being the most common so far.

There is nothing to stop anyone linking to ORCID from the web if they so choose.

For publications, it is a similar story.  They can be identified in any manner the author chooses.

For example, in https://www.gov.uk/government/uploads/system/uploads/attachment_data/file/584727/older-workers-and-the-workplace.pdf#page=9 the authors are listed as:

> Hermione Granger, Research Fellow, National Institute of Economic and Social Research (NIESR)
> Minerva McGonagall, Professor of Quantitative Social Science at UCL and NIESR Fellow
> ...

Authors may choose to use their ORCID on these documents if they wish.

Similarly, some datasets also already contain ORCID - see https://data.gov.uk/harvest/gemini-object/88a6ab08-05d1-4a91-abfa-d8c84ed540c4

This leads us to three questions.

1. Is ORCID an Open Standard?
   * Probably yes.
   * We haven't done a full assessment, but it seems to fit many of the criteria.
2. Can authors already use it?
   * Yes - on web, PDF, and print if they want.
3. Is there value in going through a more thorough assessment to *mandate* it as an Open Standard?
   * Probably not.
   * We cannot force individuals to obtain an ORCID.  It is at their discretion whether to get one and whether to use it.
   * There are no proprietary standards which are being used at the moment which should be replaced by an open one.

In summary, we are very happy for individuals to use their ORCID on published documents.  At the moment, there aren't enough use-cases for us to justify mandating this.

I'll leave this challenge open for now. If people can think of some use-cases that would make this more compelling, we'll revisit it.

Once again, thank you for raising this interesting challenge.

**Comment by oughnic** [comment]
Three questions:

1. How does this relate to the existing Government Open Standard for Exchange of contact Information?
2. How does this relate to the existing Government Open Standard for Persistent Resolveable Identifiers?
3. What competitor universal ID services are there? - Social Network IDs, National identifiers, professional registration entries come to mind.

I share Terence's concern that government documents are predominantly authored by the office, not the person, and office holders should not ordinarily become a focus.

**Comment by edent** [comment]
We had a meeting with British Library recently. They expressed some interest in ORCID, but didn't feel strongly enough about it to commit to supporting it as a challenge.  There may be interest in using DOI, which could then reopen the ORCID discussion. 

To reiterate, there's nothing stopping any author from linking to ORCID (or their blog, Twitter, etc) on official publications if they want to.

If you can find any academics working in Government who are interested in using their ORCID in their publications, we are happy to re-open this discussion.

**Comment by pigsonthewing** [comment]
> If you can find any academics working in Government who are interested in using their ORCID in their publications, we are happy to re-open this discussion.

Will the Government's Chief Scientific Adviser do?

http://orcid.org/0000-0001-7220-5273

https://en.wikipedia.org/wiki/Mark_Walport
